### DISCUSSION TITLE
There is too much log output

### DISCUSSION THREAD

**Discussion by be-hase** [JetBrains/Qodana#258] [Q&A]
I am using qodana-action.
The log output is too large, so there are often cases where it cannot be viewed from the GitHub Action log viewer.
I would like an option to adjust the log output volume.

**Comment by brichbash** [comment]
Hello, if you refer to the Gradle logs in the output, please try using the `2024.2-eap` version of the linter, i.e. `jetbrains/qodana-jvm:2024.2-eap`

**Comment by be-hase** [comment]
Wow, sounds good.
I look forward to the release of 2024.2.
